src/components/Dashboard/Toolbar.vue

```diff
@@ -15,64 +16,63 @@ const { torrentCountString, isSelectionMultiple, displayMode } = storeToRefs(das
-  { value: SortOptions.GLOBALVOLUME, title: t('dashboard.sortBy.globalVolume') },
-  { value: SortOptions.HASH, title: t('dashboard.sortBy.hash') },
-  { value: SortOptions.INFOHASH_V1, title: t('dashboard.sortBy.infohash_v1') },
-  { value: SortOptions.INFOHASH_V2, title: t('dashboard.sortBy.infohash_v2') },
-  { value: SortOptions.LAST_ACTIVITY, title: t('dashboard.sortBy.last_activity') },
-  { value: SortOptions.MAGNET_URI, title: t('dashboard.sortBy.magnet_uri') },
-  { value: SortOptions.MAX_RATIO, title: t('dashboard.sortBy.max_ratio') },
-  { value: SortOptions.MAX_SEEDING_TIME, title: t('dashboard.sortBy.max_seeding_time') },
-  { value: SortOptions.NAME, title: t('dashboard.sortBy.name') },
-  { value: SortOptions.NUM_COMPLETE, title: t('dashboard.sortBy.num_complete') },
-  { value: SortOptions.NUM_INCOMPLETE, title: t('dashboard.sortBy.num_incomplete') },
-  { value: SortOptions.NUM_LEECHS, title: t('dashboard.sortBy.num_leechs') },
-  { value: SortOptions.NUM_SEEDS, title: t('dashboard.sortBy.num_seeds') },
-  { value: SortOptions.PRIORITY, title: t('dashboard.sortBy.priority') },
-  { value: SortOptions.PROGRESS, title: t('dashboard.sortBy.progress') },
-  { value: SortOptions.RATIO, title: t('dashboard.sortBy.ratio') },
-  { value: SortOptions.RATIO_LIMIT, title: t('dashboard.sortBy.ratio_limit') },
-  { value: SortOptions.SAVE_PATH, title: t('dashboard.sortBy.save_path') },
-  { value: SortOptions.SEEDING_TIME, title: t('dashboard.sortBy.seeding_time') },
-  { value: SortOptions.SEEDING_TIME_LIMIT, title: t('dashboard.sortBy.seeding_time_limit') },
-  { value: SortOptions.SEEN_COMPLETE, title: t('dashboard.sortBy.seen_complete') },
-  { value: SortOptions.SEQ_DL, title: t('dashboard.sortBy.seq_dl') },
-  { value: SortOptions.SIZE, title: t('dashboard.sortBy.size') },
-  { value: SortOptions.STATE, title: t('dashboard.sortBy.state') },
-  { value: SortOptions.SUPER_SEEDING, title: t('dashboard.sortBy.super_seeding') },
-  { value: SortOptions.TAGS, title: t('dashboard.sortBy.tags') },
-  { value: SortOptions.TIME_ACTIVE, title: t('dashboard.sortBy.time_active') },
-  { value: SortOptions.TOTAL_SIZE, title: t('dashboard.sortBy.total_size') },
-  { value: SortOptions.TRACKER, title: t('dashboard.sortBy.tracker') },
-  { value: SortOptions.TRACKERS_COUNT, title: t('dashboard.sortBy.trackers_count') },
-  { value: SortOptions.UP_LIMIT, title: t('dashboard.sortBy.up_limit') },
-  { value: SortOptions.UPLOADED, title: t('dashboard.sortBy.uploaded') },
-  { value: SortOptions.UPLOADED_SESSION, title: t('dashboard.sortBy.uploaded_session') },
-  { value: SortOptions.UPSPEED, title: t('dashboard.sortBy.upspeed') }
-].sort((a, b) => a.title.localeCompare(b.title))
-torrentSortOptions.splice(0, 0, { value: SortOptions.DEFAULT, title: t('dashboard.sortBy.default') })
+const torrentSortOptions: {title: string, value: keyof Torrent}[] = [
+  { value: "added_on", title: t('dashboard.sortBy.added_on') },
```

Comment:
Because the qbit enum doesn't have all the fields we have, and the vuetorrent field uses integers, later in code we sort using this with a and b being `Torrent`s

```js
const { value, reverse } = sortCriterias.value.at(i++)!
const av = a[value]
const bv = b[value]
```

That's why I used `keyof Torrent` instead of an enum. Moreover, the wiki states the following: 

Parameter             | Type    | Description
----------------------|---------|------------
`sort` _optional_     | string  | Sort torrents by given key. They can be sorted using any field of the response's JSON array (which are documented below) as the sort key.

As the torrent's interface were already created, I didn't want to keep it where it was just duplicated fields.